dmoj/executors/SWIFT.py

```diff
@@ -3,9 +3,11 @@
 
 class Executor(CompiledExecutor):
     ext = 'swift'
+
```

Comment:
Do we actually want these newlines? (Or is this flake being silly)

Reply:
Many of the newlines are me appeasing flake

Reply:
Why would you need these lines? It makes no sense.

Reply:
Can we change something about a flake config option to remove the need for these lines?

Reply:
I am pretty sure it's not required except on your local machine, whose flake8 is somehow misconfigured.

Reply:
I installed flake8 with pip and ran `flake8` in the project root, I even tried maunally specifying `--config .flake8`... What would be wrong?

Reply:
Are you using the Python 2 version of flake8 or something? Because this should be impossible.

Reply:
No, I even tried re-installing with `pip3` and running with `python3 -m flake8`, but I'm still getting those errors. I could ignore them with `--ignore E302` but that won't solve the issue

Reply:
Removing the newlines and running `flake8` doesn't complain for me...